config/locales/en/requirements.yml

```diff
@@ -0,0 +1,39 @@
+en:
+  requirements:
+    title:
+      blank:
+        short_message: Enter a title
+        long_message: Enter a title
+      too_long:
+        short_message: "Enter a title that is fewer than %{max_length} characters long"
+        long_message: Enter a shorter title
```

Comment:
When you explained the long and short message they made a lot of sense but seeing them they're actually pretty confusing as often the short is longer than the long message. I assume you're also a little uncomfortable with this ;)

Of course the big challenge is what is a more appropriate name. Suggestions I have (with no presumptions they are good) are:

in_context_message / out_context_message
context_message / summary_message

with my brain trying to formulate ideas about how some messages are instant, transient, in context, in response to an initial form submission and other ones are more permanent, out of context, shown in a wider summaries. 

Don't know if that's helped at all?

Reply:
Good point - I've gone with 'form_message' and 'summary_message', which I think captures it quite well.